packages/codegen/src/babel-plugin.ts

```diff
@@ -104,27 +104,40 @@ const babelPlugin = declare((api) => {
 
           const operationOrFragmentName = createImportName(parentNode.id.name);
 
-          const filePath = state.filename;
-
-          if (!filePath) {
-            throw new Error('@data-eden/codegen: Unable to resolve filepath.');
-          }
+          if (operationOrFragmentName.indexOf('Fragment') > -1) {
+            // Replace the fragment with a Symbol
+            const replacementSymbol = t.callExpression(
+              t.identifier('Symbol.for'),
+              [t.stringLiteral(operationOrFragmentName)]
```

Comment:
let's pick some string literal so we only ever make 1 symbol rather than 1 per

`build-time-graphql-fragment` or something?

Reply:
that makes sense, will update.